### ISSUE TITLE
WASM build

### ISSUE THREAD

**Issue by amilajack** [brion/ogv.js#419]
It would be really nice if a wasm build could be provided. That would allow for a side-by-side comparison between asm.js and wasm

**Comment by brion** [comment]
Definitely! This should be possible, just haven't figured out how to hook up emscripten's wasm generation with the experimental mode in browsers yet.

**Comment by amilajack** [comment]
Also I have a question that is not related to this issue: how hard would it be to use the mkv container format in the web by compiling some library with WASM?

**Comment by brion** [comment]
mkv container format is very similar to webm (webm is technically a subset I think?) so probably feasible given an appropriate library -- but might also need codecs to go with it. Some mkv files have VP8/VP9/Vorbis/Opus, others have H.264/AAC or something else. I'm trying to avoid touching H.264, AAC, and MP3 for now (due to patent issues) but they could be done and added in as patches or plugins.

**Comment by amilajack** [comment]
Does that include mkv?
